### ISSUE TITLE
Make the Netflix subtitles hoverable so that the popup can be utilized with it.

### ISSUE THREAD

**Issue by RyanOrigens** [translate-tools/linguist#420]

**Comment by vitonsky** [comment]
What is your use case?

Linguist does not interrupt in code of specific sites. We have a similar issue about RTL where [I said about a potential problems](https://github.com/translate-tools/linguist/issues/392#issuecomment-1688634134) for this approach with "improve" site behavior.

**Comment by RyanOrigens** [comment]
Hi,

I noticed this statement on GitHub/Chrome Web Store:

"Translate web pages, highlighted text, **Netflix subtitles**, private messages, speak the translated text, and save important translations to your personal dictionary to learn words even offline."

I thought there could be a special treatment for Netflix itself, as extensions of this kind have been doing it for a while...

My proposal is to make the subtitles on Netflix selectable, so the user would be able to highlight the text and utilize the pop-up icon to translate a specific word. (or just show up the pop up with the translation instead of the icon)

Here are some examples of extensions of the same genre doing this:

Mate Translate / Reverso

For example, Mate makes the subtitles clickable for languages with spaces (Spanish, English, Portuguese, basically every single one of them). For other languages without spaces, such as Japanese or Chinese, where it may not be possible to parse the language without previous knowledge of it, you can only hover...

![image](https://github.com/translate-tools/linguist/assets/103891807/99d045a8-c9ea-46c0-8e68-7f14bd33458a)

**Comment by vitonsky** [comment]
What about translate whole page instead?
You may enable page translation and subtitles should be translated automatically. Is it works for you?

**Comment by RyanOrigens** [comment]
The whole purpose would be utilize it as a dictionary rather than a translator itself. (With the goal of acquiring a language or learning new words in a language you already know.) if this is not the goal of the project feel free to close the issue.

**Comment by vitonsky** [comment]
You're correct, the goal of Linguist both - translate content and help to learn a languages.
The only problem we have with this proposal is we have to modify a site, to implement this request.

My opinion is we must never modify a site content or change site behavior. But i see your concern about barriers to use Linguist with some sites.

Maybe we may introduce a user packages to find a tradeoff here. In this case, Linguist will not change site behavior itself, but Linguist will provide some list with packages/extensions, so users will be able to improve a behavior on specific sites they needed. User would just go to preferences page, then click something kinda "extensions" and will see a list of extensions, then will click "install"/"enable" on extensions like "netflix subtitles" and it will be installed and activated.

I'm not sure about RCE policy in Chrome store for addons with manifest V3, maybe it is not allowed, i have to research.

I will think how to resolve this problem. It probably will not be solved soon, but anyway thank you for report your use case. It is important to know what users needs.
